io.github.lemmygtk.lemoa.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "io.github.lemmygtk.lemoa",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm14"
+    ],
+    "command": "lemoa",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--env=RUST_LOG=lemoa=debug",
```

Comment:
debug log when running the flatpak, I don't think this is usefull nor correct.

Reply:
You're right, my bad.